src/utils.c

```diff
@@ -1788,6 +1788,9 @@ void utils_tidy_path(gchar *filename)
 	const gchar *needle;
 	gboolean preserve_double_backslash = FALSE;
 
+	if (utils_is_uri(filename))
+		return;
```

Comment:
@techee I was almost going to say, its ok, for URLs from schemes that are likely to be used as files the `//` is present, but on checking I found that RFC 8089 was released this month for `file:` where one difference from the previous is "The syntax given in Section 2 makes the entire authority  component, including the double slashes "//", optional.".  So I would say the rule is now starts with `file:` or contains `://`.

Yes its a less likely URL, but people dragging URLs onto Geany might get one.